src/reflow/tests.rs

```diff
@@ -719,3 +719,13 @@ of `0`.
 "###
     );
 }
+
+#[test]
+fn reflow_crlf() {
+    const INPUT: &'static str = "        /// cargo spellcheck can be configured with `-m <code>` to return a non-zero return code.\r\n        struct Foo {}";
+    println!("{}", INPUT);
```

Comment:
```suggestion
    println!("{:?}", INPUT);
```

Reply:
I deliberately used the default formatter to check if the actual string contains the indentation as I wanted it to be.

Reply:
My idea was to get escaped newlines printed, so one actually sees which newline variant fails, but either way is fine.

Reply:
Also true, makes more sense :+1: